### PR TITLE
#160577431 Setup Redux

### PR DESCRIPTION
**Pull request by IncredibleBlackMan**
#### What does this PR do?
Setup redux and other complementary dependencies

#### Description of Task to be completed?
- Simply use `yarn add redux react-redux redux-devtools` . on the existing repository.

#### How should this be manually tested?
- Clone the GitHub repository `git clone https://github.com/andela/ah-shakas-frontend` on your machine.
- Change your working directory to the project folder `cd ah-shakas-frontend`
- Checkout to this branch using the command `git checkout ch-setup-redux-160577431`
- Install the required dependencies using `yarn install`
- Run the local development server using `yarn start`

#### Any background context you want to provide?
Different directories have been created to handle actions, reducers, constants, containers, components, and the store. When the application scales, it is easier to manage these different features in their own directories than having one big module.

#### What are the relevant pivotal tracker stories?
[Setup Redux #160577431](https://www.pivotaltracker.com/story/show/160577431/comments/195865159)

### CODE REVIEW

**Review by caveinn — APPROVED** [review]
Looks good to me

**Review by michael-basweti — APPROVED** [review]
This is awesome

**Review by gloriaodipo — APPROVED** [review]
Good work @Ramon3162

**Review by koechkevin — COMMENTED** [review]
I thought we could have the default page as blank instead of the default react logo. Is it possible we have
screen shot 2018-10-30 at 11 53 54 
instead of this.
screen shot 2018-10-30 at 11 53 13

**Comment by IncredibleBlackMan** [comment]
@koechkevin makes sense. Let me make that change in a bit.

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,10 @@
...
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
+      <div className="App"></div>
```
[the remark this answers is not in the document]
Comment (Arthur236) [reply src/App.js]: kindly address this
Have it as `<div className="App"/>`

**Review by Arthur236 — APPROVED** [review]
Looks good 👍

**Pull request merged**